src/border_agent/border_agent.cpp

```diff
@@ -202,6 +203,10 @@ void BorderAgent::Start(void)
 
     mServiceInstanceName = GetServiceInstanceNameWithExtAddr(mBaseServiceInstanceName);
     UpdateMeshCopService();
+
+#if OTBR_ENABLE_EPSKC
+    otBorderAgentSetEphemeralKeyCallback(mNcp.GetInstance(), BorderAgent::HandleEpskcStateChanged, this);
+#endif
```

Comment:
To avoid duplicate callbacks, I'd suggest to put this in constructor.

Reply:
Accepted. Thanks~